redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -312,15 +312,15 @@ internal class LazyListContainerCell(
 
     if (contentView.subviews.isEmpty()) return
 
-    val content = contentView.subviews.first() as UIView ?: return
+    val content = contentView.subviews.firstOrNull() as UIView ?: return
     content.setFrame(bounds)
     contentView.setFrame(bounds)
   }
 
   override fun sizeThatFits(size: CValue<CGSize>): CValue<CGSize> {
     if (contentView.subviews.isEmpty()) return super.sizeThatFits(size)
 
-    val content = contentView.subviews.first() as UIView ?: return super.sizeThatFits(size)
+    val content = contentView.subviews.firstOrNull() as UIView ?: return super.sizeThatFits(size)
```

Comment:
```suggestion
    val content = contentView.subviews.firstOrNull() as UIView? ?: return super.sizeThatFits(size)
```

Reply:
Same, the isEmpty() check is unnecessary if we use `firstOrNull()`